rs_workflows/example.py

```diff
@@ -1,5 +1,7 @@
+"""Docstring here"""
```

Comment:
Add a real description (even if it's only one line)

Reply:
Done